test_regress/t/t_user_type_xassign.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2024 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+import vltest_bootstrap
+
+test.scenarios('simulator')
+
+test.compile()
+# test.compile(verilator_flags2=["--x-assign 0"])
```

Comment:
Did you intend to need this?, if so comment in.  If not, delete line.

Reply:
Yeah, thanks.  This was for debugging.  I'll remove it.